integration_tests/tests/cucumber.rs

```diff
@@ -3879,10 +3925,29 @@ async fn force_sync_node_with_an_army_of_pruned_nodes(
         base_node_config.force_sync_peers = get_peer_addresses(world, &peers).await.into();
         base_node_config.storage.pruning_horizon = horizon;
 
-        spawn_base_node_with_config(world, false, node_name, peers, base_node_config).await;
+        spawn_base_node_with_config(world, false, node_name, peers, base_node_config, None).await;
     }
 }
 
+#[when(expr = "I run blockchain recovery on node {word}")]
+async fn node_on_blockchain_recovery(world: &mut TariWorld, node: String) {
+    let base_node_ps = world.base_nodes.get(&node).unwrap();
+    let data_dir = base_node_ps.temp_dir_path.clone();
+
+    let base_path = data_dir.clone().into_os_string().into_string().unwrap();
+    let mut config_path = data_dir;
+    config_path.push("config.toml");
+
+    let mut cli = bn_default_cli(base_path, config_path.into_os_string().into_string().unwrap());
+
+    cli.rebuild_db = true;
+
+    let peers = base_node_ps.seed_nodes.clone();
+    let is_seed_node = base_node_ps.is_seed_node;
+
+    spawn_base_node(world, is_seed_node, node.clone(), peers, Some(cli)).await
```

Comment:
If only 1 step needs `Some(cli)` and the rest are `None` it could be better to use the `spawn_base_node_with_config` function directly and keep the `spawn_base_node` as the simple quick setup.